WalletLibrary/WalletLibrary/Extensions/Entities/IssuanceResponseContainer+WalletLibrary.swift

```diff
@@ -59,14 +59,14 @@ extension IssuanceResponseContainer: IssuanceResponseContaining {
     }
     
     private mutating func add(groupRequirement: GroupRequirement) throws {
-        try groupRequirement.validate()
+        try groupRequirement.validate().get()
```

Comment:
what does this get() do?

Reply:
it just turns a Result into the object or throws.